tests/foreman/installer/test_installer.py

```diff
@@ -227,20 +227,26 @@ def setup_capsule_repos(satellite, capsule_host, org, ak):
     else:
         # configure internal source as custom repos
         product_capsule = satellite.api.Product(organization=org.id).create()
-        for repo_variant in ['capsule', 'maintenance']:
-            dogfood_repo = dogfood_repository(
-                ohsnap=settings.ohsnap,
-                repo=repo_variant,
-                product="capsule",
-                release=settings.capsule.version.release,
-                os_release=capsule_host.os_version.major,
-                snap=settings.capsule.version.snap,
-            )
+        for repo_variant, repo_default_url in [
+            ('capsule', 'capsule_repo'),
+            ('maintenance', 'satmaintenance_repo'),
+        ]:
+            if settings.server.version.source != 'nightly':
+                dogfood_repo = dogfood_repository(
+                    ohsnap=settings.ohsnap,
+                    repo=repo_variant,
+                    product="capsule",
+                    release=settings.capsule.version.release,
+                    os_release=capsule_host.os_version.major,
+                    snap=settings.capsule.version.snap,
+                )
```

Comment:
```suggestion
            if settings.server.version.source == 'nightly':
                repo_url = url=getattr(settings.repos, repo_default_url)
            else:
                repo_url = dogfood_repository(
                    ohsnap=settings.ohsnap,
                    repo=repo_variant,
                    product="capsule",
                    release=settings.capsule.version.release,
                    os_release=capsule_host.os_version.major,
                    snap=settings.capsule.version.snap,
                ).baseurl
```

and below the `url` kwarg in `Repository` class instantiation would be `url=repo_url`.

That makes the code a bit more readable.